src/sage/rings/polynomial/multi_polynomial_sequence.py

```diff
@@ -1567,24 +1566,24 @@ def solve(self, algorithm='polybori', n=1,
         We obtain all solutions::
 
             sage: sols = S.solve(n=Infinity)                                            # needs sage.rings.polynomial.pbori
-            sage: print(reproducible_repr(sols))                                        # needs sage.rings.polynomial.pbori
-            [{x: 0, y: 1, z: 0}, {x: 1, y: 1, z: 1}]
+            sage: sols                                                                  # needs sage.rings.polynomial.pbori
+            [{z: 0, y: 1, x: 0}, {z: 1, y: 1, x: 1}]
```

Comment:
Why is this here `z < y < x` while in the other examples a bit down it is still `x < y < z`?

Reply:
Ugh, must be because I don't have the optional packages.

By the way, how can optional packages be installed in meson again? (`sage -i` doesn't work if I recalled correctly?) Find the corresponding conda package, but how do you know which conda package corresponds to which sage package?

Reply:
> Find the corresponding conda package, but how do you know which conda package corresponds to which sage package?

There are `conda.txt` files in `build/pkgs` that tell you the mapping from sage pkg to conda pkg. (But they are not actively maintained and thus may be incorrect.)

Reply:
End up most of the packages needed seems not available on conda anyway (fes ginv macaulay2 magma msolve pycryptosat) so I just try to fix it by best effort.